lib/waad.js

```diff
@@ -17,21 +17,23 @@ function Waad(options) {
   if (!this.options.accessToken) {
     throw new Error('Must supply "accessToken"');
   }
-
   if (this.options.fiddler) {
-    request = request.defaults({'proxy':'http://127.0.0.1:8888'});
+    // Fiddler proxy support temporarily disabled in v3.0.0
+    // TODO: Implement proxy support using fetch agents
+    console.warn('Fiddler proxy support is temporarily disabled in v3.0.0');

```

Comment:
Do we need to continue support this? This is a major version bump, deprecate it. Users can configure the http client however they see fit.